utils/src/main/java/org/corfudb/utils/lock/states/NoLeaseState.java

```diff
@@ -95,4 +99,14 @@ private void acquireLease() {
         }
     }
 
+    private void forceAcquire() {
+        try {
+            lockStore.forceAcquire(lock.getLockId());
+        }
+        catch (Exception e) {
+            log.error("Lock: {} could not force-acquire lease {}", lock.getLockId(), e);
```

Comment:
one extra {}

Reply:
In the case it was not able, it would still transition to HAS_LEASE, is that correct?

Reply:
I dont think so, this exception is coming from getting the lock data, so if the read fails we can not do anything

Reply:
but if attempting to forcefully acquire it throws an exception, could we affirm that it was not taken so instead it should stay in this state (NO_LEASE)?

Reply:
fixed